### ISSUE TITLE
Void methods: Wait for response?

### ISSUE THREAD

**Issue by h-huss** [jfjlaros/simpleRPC#40]
**Is your feature request related to a problem? Please describe.**
I've just noticed that calling void methods / methods without a return value does not block, they seem to return immediately. This caused errors, as I was expecting that the arduino method was done when the method returned.

**Describe the solution you'd like**
It would be nice to document that void method dont block. Alternatively, void methods could block, to behave similar to all other functions (this would be a breaking change).

**Comment by jfjlaros** [comment]
A `void` function is called just like any other function. The only difference is that no result is written after the function has finished. You can test this by adding a large `delay()` to such a function and see whether it returns directly or not.

What kind of errors are you experiencing and can you share a minimal example so I can reproduce the error?

**Comment by jfjlaros** [comment]
Indeed, the client has no way of knowing whether a `void` function has finished.

I guess we could let a `void` function write one byte to implement blocking behaviour, or we could implement it ourselves by changing the return type or, if this is not an option, by adding a `wait` function. E.g.,

On the device:
```cpp
bool wait() {
  return true;
}


void loop() {
  interface(
    Serial,
    wait, "wait: Wait for the previous command to finish. @return: True."",
    // ...
  );
}
```

and client side:
```python
mcu.longDelay()
mcu.wait()
```

I am not sure if this is a feature or a bug.

**Comment by jfjlaros** [comment]
I will give it some more thought.

I will leave this issue open as a reminder.

**Comment by jfjlaros** [comment]
Blocking `void` methods are now implemented in 66fa20a.
